trade-finance-manager-ui/server/controllers/case/parties/index.js

```diff
@@ -145,6 +147,34 @@ const postPartyDetails = (partyType) => (
       return res.redirect('/not-found');
     }
 
+    /**
```

Comment:
Can we please take into account other parties.

Reply:
done